### PR TITLE
FINERACT-1895: Fineract doesn't start with spring profile activeMqEnabled

### PR DESCRIPTION
**Pull request by hiteshprajapati**
fixing activemq config fix for in general notification

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by hiteshprajapati** [comment]
@galovics @vidakovic Please review this PR.

**Comment by vidakovic** [comment]
@hiteshprajapati I'm not sure if I folllow entirely your changes... can you explain a bit why renaming is necessary, because I general I would say using interfaces is the way to go to be able to change concrete implementations... but maybe I am missing something here. Also: please make sure to name the PR like the other ones do... correct name would be "FINERACT-1895: Fineract doesn't start with spring profile activeMqEnabled"... then everyone knows what this is about and the linking in Jira is working properly... thanks.

File: fineract-provider/src/main/resources/application.properties [hunk 1]
```diff
@@ -37,6 +37,7 @@ fineract.tenant.description=${FINERACT_DEFAULT_TENANTDB_DESCRIPTION:Default Demo
 
 fineract.mode.read-enabled=${FINERACT_MODE_READ_ENABLED:true}
 fineract.mode.write-enabled=${FINERACT_MODE_WRITE_ENABLED:true}
+fineract.mode.batch-enabled=${FINERACT_MODE_BATCH_ENABLED:true}
```
Comment (vidakovic) [inline fineract-provider/src/main/resources/application.properties:40]: @hiteshprajapati I am not so sure about this property here... why is it introduced if it's not used anywhere in your changes here? And if it's used in a different place: are we sure we want to set this by default to true? From what I can see here I don't think this property needs to be added.

**Comment by hiteshprajapati** [comment]
@vidakovic : I will raise the new PR

**Pull request closed without merging**